client/tests/webdriver/baseSpecs/insights.spec.js

```diff
@@ -17,4 +17,36 @@ describe("Insights pages", () => {
         .true
     }
   })
+
+  it("Should open each insights page for super user", async() => {
+    await Insights.openAsSuperuser()
```

Comment:
Should call `await Insights.logout()` at the end of the previous section if you want to continue the tests here as another user.